src/domains/wallet/pages/WalletDetails/hooks/use-wallet-options.ts

```diff
@@ -96,7 +96,6 @@ const getRegistrationOptions = (wallet: Contracts.IReadWriteWallet, t: TFunction
 
 	if (wallet.balance() > 0 && !wallet.isLedger() && isRestoredAndSynced(wallet)) {
 		if (
-			!isMultiSignature(wallet) &&
```

Comment:
🤔

Reply:
should be good now :) cc @samharperpittam